modules/ROOT/pages/get-started-prepare.adoc

```diff
@@ -40,10 +40,15 @@ Here's what you need in order to install Sync Gateway:
 
 Once you have all that covered ... go {get-started-install--xref} {sgw-t}.
 
+== Cluster Cluster Requirements
+Couchbase clusters intended for use with Sync Gateway must be configured with the following services:
 
+* The `Data` Service.
+* The `Index` Service.
+* The `Query` Service.
 
 [#lbl-req-minim, reftext=Minimum Requirements]
-== Couchbase Requirements
+== Couchbase Server  Requirements
```

Comment:
There are two blank characters between "Server" and "Requirements".

Reply:
Cleaned up, though this doesn't cause a problem since Asciidoc will render it as a single space when the pages are built.